configs/root.tfvars

```diff
@@ -15,7 +15,9 @@ org_network_offset  = 100
 org_network_newbits = 8    # /8 + /8 = /16
```

Comment:
this comment seems perhaps out of date?

Reply:
I don't know, seem right to me, but I'm not fully following it. It says to me we are starting with a CIDR of 10/8 and creating networks of 10.x/16 starting with 10.100/16, which is what, in fact, we are doing.